wwf/basics/interp.py

```diff
@@ -0,0 +1,18 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/05_basics.interp.show_at.ipynb (unless otherwise specified).
+
+__all__ = []
+
+# Cell
+@patch
+def __getitem__(self:Interpretation, idxs):
+    "Get the inputs, preds, targets, decoded outputs, and losses at `idx`"
+    if not is_listy(idxs): idxs = [idxs]
+    attrs = 'inputs,preds,targs,decoded,losses'
+    res = L([getattr(self, attr)[idxs] for attr in attrs.split(',')])
+    return res
+
+@patch
+def show_at(self:Interpretation, idx:int, **kwargs):
```

Comment:
Perhaps include a docstring here. Also would we want to have them be able to show multiple index's? Or is just one good enough. I'm leaning towards a situation similar to `__getitem__` where we check for a list of index's

Reply:
Sure! I assume that the creation of the grid for showing multiple items would be delegated to `show_results`, is that correct?

Reply:
Yes that would be (sorry for just now getting to that)